### DISCUSSION TITLE
Suggestion: Button to 'apply to all roads in this junction'

### DISCUSSION THREAD

**Discussion by add1ct3dd** [slyh/Cities2-TrafficLightsEnhancement#39] [Ideas]
Hey,

So I came across the feature via a YouTube video and it looks fantastic!!

The only thing I'd love to see is most of my junctions are + junctions that offer the same road functionality, I'd love to be able to set on one incoming road the rules that 'apply' and then apply it to the entire junction if that makes sense? This would make setting up my roads so much faster. It may already be on your list of ideas, but if not here you go!

For example a 6 lane road + junction split with 3 lanes on each side; one lane goes left, one middle, one right. Set this once and then click a button for it to instantly apply to the other 3 incoming roads.

Thanks for your time :)

**Comment by slyh** [comment]
It is technically possible to achieve such functionality, but I have to prioritise implementing major features first.
